### PR TITLE
Adding Server/resources/config.json missing configurations docs to server.md

### PR DESCRIPTION
**Pull request by Khaikaa**
Adding all missing configurations in server.md and clarifying some others a little bit. server.md should be complete now.

### CODE REVIEW

**Review by blinkuz — APPROVED** [review]

**Comment by erjdev** [comment]
Is this PR waiting on any additional review, or may it be merged?

**Comment by pandinocoder** [comment]
It was waiting on a review from myself, though... I didn't know the PR was open. 😃  Reviewing now and merging.

**Pull request merged**
